tests/rustfmt/main.rs

```diff
@@ -174,3 +174,20 @@ fn rustfmt_emits_error_on_line_overflow_true() {
         "line formatted, but exceeded maximum width (maximum: 100 (see `max_width` option)"
     ))
 }
+
+#[test]
+fn rustfmt_emits_error_on_line_overflow_true_when_macro_formatting_fails() {
+    // See also https://github.com/rust-lang/rustfmt/issues/5700
+    let args = [
+        "--config-path",
+        "tests/config/issue_5700.toml",
+        "--config",
+        "error_on_line_overflow=true",
```

Comment:
If we're using a config file already should we just set `error_on_line_overflow=true` there? Although part of me thinks that for clarity we don't need the config file since the only configs that are relevant here are `error_on_line_overflow=true` and `max_width=120`, which we could pass to the test like:

```suggestion
        "--config",
        "error_on_line_overflow=true,max_width=120",
```